src/common/Square.tsx

```diff
@@ -1,22 +1,29 @@
 import styled from "styled-components";
```

Comment:
Just curious, how did you go about selecting this library for styling? First time I'm seeing it

Reply:
I used this library at my last position. I think the biggest pro is its dynamic styling. You can pass in props and use JS expressions to style accordingly. I haven't explored many libraries, so I'm not sure if others have it, but styled-components lets you use the `as` keyword to give even more flexibility. For example, you can render a component `as` a `button` or `div` based on a condition you specify.